src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -2265,7 +2265,7 @@ private function class_name_updates_to_attributes_updates() {
 	 * @param int $shift_this_point Accumulate and return shift for this position.
 	 * @return int How many bytes the given pointer moved in response to the updates.
 	 */
-	private function apply_attributes_updates( $shift_this_point = 0 ) {
+	private function apply_attributes_updates( $shift_this_point ) {
```

Comment:
This private function was used in _exactly_ 1 place and the argument was always provided.